habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml

```diff
@@ -41,7 +41,8 @@ habitat_baselines:
   evaluator:
     _target_: habitat_baselines.rl.multi_agent.habitat_mas_evaluator.HabitatMASEvaluator
 
-  image_dir: "video_dir/image_dir/"
+  image_dir: "video_multi/spot_drone/image_dir/"
+  video_dir: "video_multi/spot_drone"
```

Comment:
Please disable image saving by default, it is very slow and not desired by everyone. Simply commented out this line and  add a comment for this option.

Reply:
The path doesn't affect whether it saves images. It was set not to save images if we just leave `image_option` blank.
https://github.com/SgtVincent/habitat-lab/blob/2a7b9c7c433d58d5e6dc96bfe1979344edf581c5/habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml#L49-L50